sql/sql_admin.cc

```diff
@@ -1728,8 +1751,24 @@ class Alter_instance_reload_tls : public Alter_instance {
   LEX_CSTRING channel_name_;
   bool force_;
   Ssl_acceptor_context_type context_type_;
+  static std::vector<ssl_reload_callback_t> callbacks_;
```

Comment:
Do we really need to keep more than one active `ssl_reload_callback_t` here?

Reply:
This change extends the plugin interface. Currently, only the X plugin uses it, ut in theory, others could also implement this. 

For example, we already got a ticket for PXC to support alter instance reload tls for the wsrep port, which could use this extension point.